### ISSUE TITLE
Check PyData sphinx theme/Hugo theme for Spec3 Guidelines

### ISSUE THREAD

**Issue by juanis2112** [scientific-python/specs#252]
It would be good to mention what projects are actually using for accessibility like the themes. Starting with these two themes we should probably check whether they meet our accessibility recommendations and write it up as part of the spec.

**Comment by trallard** [comment]
This is work that falls under the current. Scientific Python grant https://github.com/Quansight-Labs/czi-scientific-python-mgmt

Considering that the work is already tracked elsewhere and worked on, is there need to keep this issue open?

**Comment by tupui** [comment]
Yes because we would still want to get recommendations (from the outcome of the grant or not) into a SPEC
